Nomster/RESTLoader.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  RESTLoader.swift
+//  Nomster
+//
+//  Created by Klaus Meyer on 06.03.15.
+//  Copyright (c) 2015 Tobias Scholze. All rights reserved.
+//
+
+import Foundation
+
+class RESTLoader {
+
+    let API_BASE = "http://localhost:3000"
```

Comment:
Not really. In a later stage, we should extract all configs to a settings bundle. You can access them via the settings app on your iOS device.